ui/address/contract/methods/form/resultPublicClient/ItemPrimitive.tsx

```diff
@@ -54,7 +55,7 @@ const ItemPrimitive = ({ abiParameter, data, level, hideLabel }: Props) => {
     if (intMatch && typeof data === 'bigint' && intMatch.max > INT_TOOLTIP_THRESHOLD && data > INT_TOOLTIP_THRESHOLD) {
       const dividedValue = BigNumber(data.toString()).div(WEI);
       return (
-        <Tooltip label={ dividedValue.toLocaleString() + ' ETH' }>
+        <Tooltip label={ dividedValue.toLocaleString() + ' ' + currencyUnits.ether }>
```

Comment:
Generally, this value does not represent an amount of native currency. It could be any arbitrary number, such as the "total supply" of a token. So, I think using the native currency symbol here would be incorrect.